bakery/src/scripts/link_single.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
 exec 2> >(tee "$LINKED_OUTPUT/stderr" >&2)
-
-link-single "$BAKED_INPUT" "$BAKED_META_INPUT" "$(cat "${BOOK_INPUT}"/slug)" "$FETCHED_INPUT/book-slugs.json" "$LINKED_OUTPUT/$(cat "$BOOK_INPUT"/slug).linked.xhtml"
+if [[ -z "${TARGET_BOOK}" ]]; then
```

Comment:
I was quickly testing these out, and I don't think this works currently in the manner intended. I believe to cause the `cp` short circuit this should be `-n` based on the convention of passing a value if `--single` is used.

Reply:
yes that was a mistake, thanks